Documentation/configure-kubectl.md

```diff
@@ -20,13 +20,13 @@ $ mv kubectl /usr/local/bin/kubectl
 
 Configure your local Kubernetes client using the following commands:
 
-* Replace `${MASTER_IP}`
+* Replace `${MASTER_HOST}`
```

Comment:
Can you better explain what MASTER_HOST is, given that it is actually a `<host>[:<port>]` instead of simply an IP now